pilot/pkg/model/jwks_resolver_test.go

```diff
@@ -144,13 +151,20 @@ func TestGetPublicKeyReorderedKey(t *testing.T) {
 	}
 	for _, c := range cases {
 		pk, err := r.GetPublicKey(c.in[0], c.in[1])
+		key := jwtKey{issuer: c.in[0], jwksURI: c.in[1]}
+		time.Sleep(time.Millisecond * 100)
```

Comment:
can we avoid sleeping? https://github.com/istio/istio/wiki/Test-Flakes#timing-dependencies

Reply:
In middle of changes, will remove this soon.